### PR TITLE
Starlight CollatorConfiguration pallet and basic integration tests

### PR DESCRIPTION
**Pull request by fgamundi**
Adds Tanssi's pallet_configuration to Starlight runtime as `CollatorConfiguration`.
Also set's up basic integration tests for the runtime

### CODE REVIEW

File: solo-chains/runtime/starlight/src/lib.rs [hunk 1]
```diff
@@ -1382,6 +1385,45 @@ impl OnSwap for SwapLeases {
...
+impl tp_traits::GetSessionIndex<u32> for CurrentSessionIndexGetter {
+    /// Returns current session index.
+    fn session_index() -> u32 {
+        Session::current_index()
+    }
+}
+
+impl pallet_configuration::Config for Runtime {
+    type SessionDelay = ConstU32<2>;
+    type SessionIndex = u32;
```
Comment (girazoki) [inline solo-chains/runtime/starlight/src/lib.rs:1399]:
```suggestion
    type SessionIndex = SessionIndex;
```

File: solo-chains/runtime/starlight/src/lib.rs [hunk 2]
```diff
@@ -1382,6 +1385,45 @@ impl OnSwap for SwapLeases {
...
+    type CurrentSessionIndex = CurrentSessionIndexGetter;
+    type AuthorityId = BeefyId;
+    type WeightInfo = ();
+}
+
+parameter_types! {
+    // pub const DepositAmount: Balance = 100 * UNIT;
+    pub const MaxLengthTokenSymbol: u32 = 255;
+}
+// impl pallet_registrar::Config for Runtime {
```
Comment (girazoki) [inline solo-chains/runtime/starlight/src/lib.rs:1409]: remoe this, @tmpolaczyk will add it

**Review by girazoki — APPROVED** [review]
small comments, but looks good!

**Review by akramhussein — APPROVED** [review]

File: solo-chains/runtime/starlight/tests/common/mod.rs [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+#![allow(dead_code)]
```
Comment (akramhussein) [inline solo-chains/runtime/starlight/tests/common/mod.rs:16]: I think there should be a new line here?

**Pull request merged**